measures/2019/measures-data.json

```diff
@@ -20859,7 +20858,6 @@
     "submissionMethods": [
       "claims",
       "electronicHealthRecord",
-      "cmsWebInterface",
```

Comment:
Do not remove `cmsWebInterface` as a valid submission method. WI _will_ still submit this measure, but it will get the denominator reduction for `QUALITY-065`

Reply:
K. Originally wasn't sure and asked Shad. He wasn't sure either so I left it in PR review just incase and was going to ask about it again today. Thanks for the clarifications.